quickstart.yml

```diff
@@ -0,0 +1,77 @@
+###########################################################################
+#######             FOR DEMONSTRATION PURPOSES ONLY                 #######
+###########################################################################
+
+version: '3'
+
+services:
+
+  kratos-migrate:
+    build:
+      context: .
+      dockerfile: Dockerfile-build
+    environment:
+      - DSN=sqlite://var/lib/sqlite/db.sqlite
+    volumes:
+      - kratos-sqlite:/var/lib/sqlite/
+    command:
+      migrate sql -e --yes
+    restart: on-failure
+    network:
+      - intranet
+
+  kratos-selfservice-ui:
+    image: oryd/kratos-selfservice-ui-node:v0.0.24
+    ports:
+      - "4435:4435"
+    environment:
+      - PORT=4435
+      - KRATOS_PUBLIC_URL=http://kratos:4433/
+      - KRATOS_BROWSER_URL=http://127.0.0.1:4433/
+    network:
+      - intranet
+
+  kratos:
+    depends_on:
+      - kratos-migrate
+      - kratos-seflservice-ui
```

Comment:
typo - `kratos-seflservice-ui` it should be `kratos-selflservice-ui`

Reply:
thank you! :)